Assets/Scenes/GameBoardMain/Scripts/InstantiateGemPrefabs.cs

```diff
@@ -13,9 +17,18 @@ void Start()
     	Debug.LogFormat("Board first row: [{0}, {1}, {2}, {3}, {4}]",
     		board.GemAt(0, 0), board.GemAt(1, 0), board.GemAt(2, 0), board.GemAt(3, 0), board.GemAt(4, 0));
 
-        for (int i = 0; i < 10; i++)
+        instantiateGems();
+    }
+
+    private void instantiateGems()
+    {
+        var cols = 5;
+        var rows = 5;
+        var max = 1;
+        List<List<Array>> matrix = CoordinateSystem.GetCoordinates.Coordinates(cols, rows, max);
+        foreach (var coordinateArray in matrix)
         {
-            Instantiate(Prefab, new Vector3(i * 2.0F, 0, 0), Quaternion.identity);
+            Instantiate(Prefab, new Vector3(1 * 2.0F, 0, 0), Quaternion.identity);
```

Comment:
I still need to access the locations from the coordinateArray. Will do that today. Was tired of looking at my screen last night.